Src/Extern/HYPRE/AMReX_Habec_2D_K.H

```diff
@@ -0,0 +1,778 @@
+#ifndef AMREX_HABEC_2D_H_
+#define AMREX_HABEC_2D_H_
+
+#include <AMReX_LO_BCTYPES.H>
+#ifdef AMREX_USE_HYPRE
+#include <AMReX_Hypre.H>
+#endif
+#include <AMReX_MultiFabUtil.H>
+
+#ifdef AMREX_USE_EB
+#include <AMReX_EBMultiFabUtil.H>
+#include <AMReX_MultiCutFab.H>
+#include <AMReX_EBFabFactory.H>
+#include <AMReX_MLEBABecLap_K.H>
+#endif
+
+namespace amrex {
+
+AMREX_FORCE_INLINE
+void amrex_hpacoef (Box const& box, 
```

Comment:
This 2d version of `amrex_hpacoef` is identical to the 3d version.  Could we move them to a new file `AMReX_Habec_K.H`?  Maybe move the following block in `AMReX_HypreABecLap.cpp` to `AMReX_Habec_K.H` too?

```
#if (AMREX_SPACEDIM == 2)                                                                                                                                                                                  
#include <AMReX_Habec_2D_K.H>
#elif (AMREX_SPACEDIM == 3)
#include <AMReX_Habec_3D_K.H>
#endif
```

Reply:
sure!!